guide/english/svg/index.md

```diff
@@ -129,6 +129,9 @@ This draws the following chart:
 
 [Browser support for SVG](https://caniuse.com/#feat=svg) is available in all modern browsers. There are some issues with scaling in IE 9 through IE 11 however they can be overcome with the use of the `width`, `height`, `viewbox`, and CSS.
 
+### Responsive SVG
+SVGs will not always have a clearly defined aspect ratio. By default, SVGs will be drawn in accordance with the dimensions defined in the code. Typically, the height and width attributes can be removed from the code since the size of the SVG will be defined by the viewBox. The preserveAspectRatio attribute can be added to an SVG to dictate how the element will resize in a viewport with a different aspect ratio.
```

Comment:
```suggestion
SVGs will not always have a clearly defined aspect ratio. By default, SVGs will be drawn in accordance with the dimensions defined in the code. Typically, the height and width attributes can be removed from the code since the size of the SVG will be defined by the viewBox. The `preserveAspectRatio` attribute can be added to an SVG to dictate how the element will resize in a viewport with a different aspect ratio.
```

Reply:
Please make the attribute in code form as recommended